geomesa-tools/bin/geomesa-accumulo

```diff
@@ -0,0 +1,251 @@
+    GEOMESA_CONF_DIR=${GEOMESA_HOME}/conf
+elif [[ $1 = configure ]]; then
+    message="Warning: GEOMESA_CONF_DIR was already set, probably by a prior configuration."
+    message="${message}\n The current value is ${GEOMESA_CONF_DIR}."
+    echo >&2 ""
+    echo -e >&2 "$message"
+    echo >&2 ""
+    read -p "Do you want to reset this to ${GEOMESA_HOME}/conf? Y\n " -n 1 -r
+    if [[  $REPLY =~ ^[Yy]$ ]]; then
+        GEOMESA_CONF_DIR=${GEOMESA_HOME}/conf
+        echo >&2 ""
+        echo "Now set to ${GEOMESA_CONF_DIR}"
+    fi
+    echo >&2 ""
+fi
+
+GEOMESA_OPTS="${GEOMESA_OPTS} -Dlog4j.configuration=file://${GEOMESA_CONF_DIR}/log4j.properties"
+
+
+# ACCUMULO paths, user can hard set these, or rely on this script to find set them via
+if [ -n "$ACCUMULO_HOME" ]; then
+    if [ -z "$ACCUMULO_LIB" -a -d "${ACCUMULO_HOME}/lib" ]; then
+        ACCUMULO_LIB=${ACCUMULO_HOME}/lib
+    fi
+    if [ -z "$ACCUMULO_CONF_DIR" -a -d "${ACCUMULO_HOME}/conf" ]; then
+        ACCUMULO_CONF_DIR=${ACCUMULO_HOME}/conf
+    fi
+fi
+
+# HADOOP paths, user can hard set these, or rely on this script to find set them via HADOOP_HOME
+if [ -n "$HADOOP_HOME" ]; then
+    if [ -z "$HADOOP_CONF_DIR" -a -d "${HADOOP_HOME}/etc/hadoop" ]; then
+        HADOOP_CONF_DIR=${HADOOP_HOME}/etc/hadoop
+    fi
+fi
+
+# Warn user about missing environment variables
+if [[ (-z "$ACCUMULO_HOME") || (-z "$HADOOP_HOME") ]]; then
+    message="Warning: you have not set ACCUMULO_HOME and/or HADOOP_HOME as environment variables."
+    message="${message}\nGeoMesa tools will not run without the appropriate Accumulo and Hadoop jars in the tools classpath."
+    message="${message}\nPlease ensure that those jars are present in the classpath by running 'geomesa classpath' ."
+    message="${message}\nTo take corrective action, please place the necessary jar files in the lib directory of geomesa-tools."
+    echo >&2 ""
+    echo -e >&2 "$message"
+    echo >&2 ""
+elif [[ (-z "$ACCUMULO_CONF_DIR") || (-z "$HADOOP_CONF_DIR")]]; then
+    message="Warning: you have not set ACCUMULO_CONF_DIR and/or HADOOP_CONF_DIR as environment variables."
+    message="${message}\nGeoMesa tools looks for these folders relative to the paths of the ACCUMULO_HOME and HADOOP_HOME environment variables."
+    message="${message}\nGeoMesa tools can run without these variables set, however the user will be required to provide the additional instanceId and zookeepers parameters."
+    message="${message}\nTo take corrective action, please set these environment variable to the appropriate locations"
+    echo >&2 ""
+    echo -e >&2 "$message"
+    echo >&2 ""
+fi
+
+function findJars() {
+    home="$1"
+    CP=""
+    if [ -n "$home" -a -d "$home" ]; then
+        for jar in $(find ${home} -name "*.jar"); do
+            if [[ ("$2" != "true") || (("$jar" != *"test"*) && ("$jar" != *"slf4j"*)) ]]; then
+                if [ "$jar" = "${jar%-sources.jar}" -a "$jar" = "${jar%-test.jar}" ]; then
```

Comment:
This is to verify that it's not a -test or -sources jar. `${foo%suffix}` will extract the suffix from foo, so:
```bash
$ foo="test.jar"
$ echo ${foo%-sources.jar}
test.jar
$ foo="test-sources.jar"
$ echo ${foo%-sources.jar}
test
```
if foo == foo with the -sources (and -test) stripped off, then it doesn't end in -sources and it's a legit jar.